torch/_torch_docs.py

```diff
@@ -9216,6 +9216,13 @@ def merge_dicts(*dicts):
 
 The shape of the tensor is defined by the variable argument :attr:`size`.
 
+.. note::
+    For complex dtypes, the tensor is i.i.d. sampled from a `complex normal distribution`_ with zero mean and
+    unit variance as :math:`\text{{out}}_{{i}} \sim \mathcal{{CN}}(0, 1)`. This is equivalent to
+    :math:`\mathcal{{R}}(\text{{out}}_{{i}}) \sim \mathcal{{N}}(0, \frac{{1}}{{2}})` and
```

Comment:
Mind writing the real part as `\operatorname{Re}` and the imaginary part as `\operatorname{Im}`?